src/init-action.ts

```diff
@@ -201,7 +203,12 @@ async function run() {
     const codeqlRam = process.env["CODEQL_RAM"] || "6500";
     core.exportVariable("CODEQL_RAM", codeqlRam);
 
-    const tracerConfig = await runInit(codeql, config);
+    const sourceRoot = path.join(
```

Comment:
Using `resolve` has the advantage that it "does the right thing" if someone specifies an absolute path for the source root.

```suggestion
    const sourceRoot = path.resolve(
```